package.json

```diff
@@ -133,6 +135,16 @@
         "title": "Expand Alias",
         "category": "PowerShell"
       },
+      {
+        "command": "PowerShell.SetISECompatibility",
+        "title": "Set ISE like compatibility",
+        "category": "PowerShell"
+      },
+      {
+        "command": "PowerShell.UnsetISECompatibility",
+        "title": "Unset ISE like compatibility (restore to defaults)",
```

Comment:
I was thinking

"Disable ISE Mode (restore default settings)"

What do you think of that?

Reply:
copy pasta: That sounds better than what I originally had. My original was "Make ISE-like" and "Make Default-like" so enable and disable is much better.